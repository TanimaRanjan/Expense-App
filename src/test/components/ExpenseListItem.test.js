import React from 'react'
import ExpenseListItem from '../../components/ExpenseListItem'
import expenses from '../fixtures/expenses'
import {shallow} from 'enzyme'

test('Should render Expense List Item ', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]}/>)
    expect(wrapper).toMatchSnapshot()
})