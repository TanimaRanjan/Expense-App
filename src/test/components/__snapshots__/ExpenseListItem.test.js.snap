// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Expense List Item  1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      Rent1
    </h3>
  </Link>
  <p>
    195
     - 
    <Component />
  </p>
</div>
`;
